registry/validate_doc_update.js

```diff
@@ -371,10 +371,6 @@ module.exports = function (doc, oldDoc, user, dbCtx) {
 
   function checkDep(version, dep, t) {
     ridiculousDeps()
-    if (!entity) {
```

Comment:
I'm concerned that this is doing more than it appears? Do you know what `entity` is here and why it might be unset?